extensions/amazon-lambda-resteasy/runtime/pom.xml

```diff
@@ -25,6 +25,12 @@
         <dependency>
             <groupId>com.amazonaws.serverless</groupId>
             <artifactId>aws-serverless-java-container-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Hmmm, don't we need to add the corresponding JBoss Logging adapter every time we add an exclusion of this sort?

Reply:
Oops, forgot that one. Thanks!